### PR TITLE
Issues/6

### PR DESCRIPTION
**Pull request by lkrishnamurthy**
Fixed issues #6  and #7

### CODE REVIEW

File: .env-skit [hunk 1]
```diff
@@ -0,0 +1,36 @@
+USE_WEBUI=true
+ALCHEMY_API_KEY=a19a44dd53585567b29bbf17689ac5c23318e3cc
```
Comment (kognate) [inline .env-skit]: I don't think you want this api key here.

**Review by kognate — CHANGES_REQUESTED** [review]

File: .env-skit (hunk 1 above)
Comment (lkrishnamurthy) [reply .env-skit]: No. You are right. The .gitignore missed it and I don't remember checking it in. Thanks for catching it.

Comment (lkrishnamurthy) [reply .env-skit]: Removed .env-skit.

**Comment by germanattanasio** [comment]
@lkrishnamurthy you need to remove the key from the git history. see https://help.github.com/articles/remove-sensitive-data/

**Review by kognate — APPROVED** [review]
Looks good.

**Pull request merged**